src/test/ui/traits/negative-impls/negative-impl-sized.rs

```diff
@@ -1,9 +1,10 @@
 #![feature(negative_impls)]
 
-// Test a negative impl for a trait requires `T: ?Sized`.
+// Test that negative impls for a trait requires `T: ?Sized`.
 
 trait MyTrait {}
 
-impl<T> !MyTrait for T {} //~ ERROR auto traits must not contain where bounds
+impl<T> !MyTrait for T {}
+//~^ ERROR negative impls on type parameters must not contain where bounds
```

Comment:
is there any way to highlight the `T: Sized` bound? e.g., [for drop we print something like](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=2f9961a369021a2a2d68d91c67e1456d)

> error[E0367]: `Drop` impl requires `T: std::marker::Sized` but the struct it is implemented for does not